website/blog/2019-01-00-1.16.0.md

```diff
@@ -12,61 +12,213 @@ This release adds TODO.
 
 ### HTML
 
-#### Preserve surrounding linebreaks ([#5596] by [@ikatyang])
-
-TODO
+#### Respect surrounding linebreaks ([#5596] by [@ikatyang])
+
+Previously, Prettier always put elements in a single line if they fit the print
+width, but it is not suitable for cases that is used as lists or `if-else`
+blocks. To solve this problem, we'll respect surrounding linebreaks for all
+elements since there is no reliable way to determine if it's used as lists.
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div class="list">
+  <div class="item">Jan</div>
+</div>
+<a v-if="i !== -1" href="#" @click.prevent="select(i)">
+  {{ i }}
+</a>
+<span v-else>
+  <slot name="ellipsis">&hellip;</slot>
+</span>
+
+<!-- Output (Prettier 1.15) -->
+<div class="list"><div class="item">Jan</div></div>
+<a v-if="i !== -1" href="#" @click.prevent="select(i)"> {{ i }} </a>
+<span v-else> <slot name="ellipsis">&hellip;</slot> </span>
+
+<!-- Output (Prettier 1.16) -->
+<div class="list">
+  <div class="item">Jan</div>
+</div>
+<a v-if="i !== -1" href="#" @click.prevent="select(i)">
+  {{ i }}
+</a>
+<span v-else>
+  <slot name="ellipsis">&hellip;</slot>
+</span>
+```
 
 ### General
 
 #### Better CRLF handling ([#5494] by [@ikatyang])
 
-TODO
+You might have noticed a few weird things happened only on Windows, which is
+usually caused by the bad CRLF handling. In this release, we completely get rid
+of them by normalizing linebreaks before/after formatting.
```

Comment:
Is this an internal change, or will it make the CRLF story better for users? Since I'm confused, maybe we shouldn't mention this one if it's an internal change?

Reply:
#5494 is the PR in question here.

Reply:
This PR did affect the output so I think it should be mentioned somewhere, though the issue it fixed were never reported (e.g., the mixed line endings caused by `// prettier-ignore`).